ssh-server-config/set-login-grace-time-gdcv.yaml

```diff
@@ -0,0 +1,71 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ssh-server-config
+  namespace: kube-system
+  labels:
+    app: ssh-server-config
+spec:
+  selector:
+    matchLabels:
+      app: ssh-server-config
+  template:
+    metadata:
+      labels:
+        app: ssh-server-config
+    spec:
+      hostPID: true
+      tolerations:
+        - key: node-role.kubernetes.io/control-plane
```

Comment:
The security incident exists in older version of clusters, so we may want to tolerate the old master taint as well.

Actually, we may just tolerate all taints.

Reply:
+1

Reply:
Changed to tolerate all taints